extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -125,7 +125,9 @@ protected function requestParsoid( $method, $title, $params ) {
 			}
 			\Wikia\Logger\WikiaLogger::instance()->error( 'ApiVisualEditor_requestParsoid', [
 				'method' => $method,
-				'error' => 'parsoidserver-' . $code
+				'error' => 'parsoidserver-' . $code,
```

Comment:
error_code may be missleading in my opinion as $code in this place has always value 'http-bad-status'